Artifacts/PgSql/Structure/Security/0050-Tables - RLPhase2.sql

```diff
@@ -0,0 +1,130 @@
+-- SPDX-License-Identifier: Apache-2.0
```

Comment:
Script should be renamed... TBD.

Reply:
0050-Tables-MultipleAuthStrats.sql updated